docker-compose.yml

```diff
@@ -5,10 +5,6 @@ services:
     image: diffy
     ports:
       - 3000:3000
-    volumes:
```

Comment:
I added this named volumes because I sort of prefer them rather than having anonymous volumes scattered around. I realise that you need to create these before hand to make it possible for the command to run so that's a downside. I'm hesitant to remove this unless you have a compelling reason.

Reply:
My understanding is that you need volumes when you need to share data, between host and container. 

In this case, I can see sharing `node_modules` being useful, while developing (i.e. debugging inside packages), but I feel this can be made part of a development `override` docker-compose file.